modules/ROOT/pages/architecture/hexagonale_architecture.adoc

```diff
@@ -194,6 +194,169 @@ This makes the application more difficult to understand and harder to locate the
+
+For larger applications, it makes sense to additionally implement the division using Maven or Gradle modules.
+This has the advantage that the application core of the hexagon remains completely without special technical frameworks such as JPA.
+As a result, it is technically not possible to use these functions in the core, which permanently leads to a "cleaner" core.
+
+//[[img-t-hexagonal-architecture-module]]
+//.Module-based hexagonal component architecture
+//image::hexagonal_component_architecture_modules.drawio.svg["devonfw hexagonal architecture module",scaledwidth="80%",align="center"]
+
+The division into modules looks like this:
+
+[source,plaintext]
+----
+application A1
+├── bootstrap (module)
+└── component C1 (module)
+    ├── core (module)
+    │   ├── domain
+    │   ├── port
+    │   │   ├── in
+    │   │   └── out
+    │   ├── usecase
+    │   └── service (optional)
+    └── adapter (module)
+        ├── in
+        │   └── web
+        │       └── (analog with packages)
+        └── out
+            └── repository
+                └── (analog with packages)
+----
+
+At the main level of the application there is the Bootstrap Module, which in Spring contains, for example, the configurations and SpringBootApplication.
+This module provides the starting point of the application and at the same time has a dependency on all cores and adapters of the business components.
+The business components, of which there can be any number, are also defined as modules at the main level.
+In the first step, we only assume one business module.
+This is divided into both the core module and the adapter module.
+In this way, core and adapter can define different dependencies and the core remains free of technical frameworks.
+
+The adapters in turn are divided into `in` and `out` adapters as in the package structure described above.
+However, no division into modules is recommended here, but instead exclusively into packages.
+A division here into modules increases the complexity significantly and also makes the expansion of new adapters much more complex, as these also have to be made known to the bootstrap module.
+With a large number of adapters and a high number of different dependencies, a tradeoff is to further divide the in and out adapters into modules.
+However, the recommendation here is to initially use packages.
+
+The management of dependencies should aim to define them as far "down" as possible.
+This means in particular that technical dependencies are always defined in the adapters.
+In order to keep versions consistent if possible, dependency management in the master pom can be used in Maven.
+The advantage of this is that the version only needs to be defined at the main level and does not need to be specified in every adapter pom.
+
+For internal dependencies it is necessary that the adapter has access to the core.
```

Comment:
Internal dependencies sounds like dependencies betweeen components. Maybe it would be better to explain this a bit further. Like "It is necessary that the adapters can access the core, so that incoming adapters can call core logic and the dependecy inversion mentioned earlier between outbound adapters and core can be implemented. hence there need to be a dependency between adapter module and the core module."